site/en/docs/extensions/whatsnew/index.md

```diff
@@ -3,18 +3,29 @@ layout: 'layouts/doc-post.njk'
 title: What's new in Chrome extensions
 description: 'Recent changes to the Chrome extensions platform, documentation, and policy'
 date: 2021-02-25
-updated: 2022-09-27
+updated: 2022-09-28
 
 # Note: disabling the linter for duplicate headings because this isn't hierarchical and it needs
 # smaller font headings.
-
 ---
 <!--lint disable no-duplicate-headings-->
 <!--lint disable first-heading-level-->
 
 Check this page often to learn about changes to the Chrome extensions platform, its documentation,
 and related policy or other changes.
 
+### Blog post: More details on the transition to Manifest V3 {: #blog-mv3-transition-details }
+
+September 28, 2022
+
+[This blog post](/blog/more-mv2-transition/) provides clarifications on the Manifest V2 deprecation timeline. The [Manifest V2 support timeline](/docs/extensions/mv3/mv2-sunset/) has also been updated to reflect this information.
+
+### Docs update: Known issues when migrating to Manifest V3 {: #known-issues }
+
+September 28, 2022
+
+[This new article](/docs/extensions/mv3/intro/) provides an overview of major features currently in development and open bugs.
```

Comment:
This isn't a new article. Is this the right link?

Reply:
Oops, wrong page! I meant to link to the known issues page. 

```suggestion
[This new article](/docs/extensions/mv3/known-issues/) provides an overview of major features currently in development and open bugs.
```